conf/repos.yaml.template

```diff
@@ -7,29 +7,31 @@ REPOS:
   RHEL6_REPO: '@format {this[repos].rhel_repo_host}/pub/rhel6.repo'
   RHEL7_REPO: '@format {this[repos].rhel_repo_host}/pub/rhel7.repo'
   RHEL8_REPO: '@format {this[repos].rhel_repo_host}/pub/rhel8.repo'
+  SATELLITE6_REPO: replace-with-repo-link
   # Provide link to rhel6/7/8 repositories URL as we need all OS packages in order
   # to have real installation media for provisioning procedure
-  # RHEL6_OS: replace-with-rhel6-os-http-link
-  # RHEL7_OS: replace-with-rhel7-os-http-link
-  # RHEL8_OS:
-      # BASEOS: replace-with-rhel8-os-baseos-http-link
-      # APPSTREAM: replace-with-rhel8-os-appstream-http-link
-  # RHEL7_OPTIONAL: replace-with-rhel7-optional-url
-  # RHEL7_EXTRAS: replace-with-rhel7-extras-url
+  RHEL6_OS: replace-with-rhel6-os-http-link
+  RHEL7_OS: replace-with-rhel7-os-http-link
+  RHEL8_OS:
+    BASEOS: replace-with-rhel8-os-baseos-http-link
+    APPSTREAM: replace-with-rhel8-os-appstream-http-link
+  RHEL7_OPTIONAL: replace-with-rhel7-optional-url
+  RHEL7_EXTRAS: replace-with-rhel7-extras-url
   # If capsule and satellite tools repositories available related packages will
   # be pulled from there instead of using the CDN channel. These information is
   # more suited to be used for downstream, downstream-iso and zstream builds.
-  # CAPSULE_REPO: replace-with-capsule-http-link
-  # SATTOOLS_REPO:
-      # RHEL6: replace-with-rhel6-http-link
-      # RHEL7: replace-with-rhel7-http-link
-      # RHEL8: replace-with-rhel8-http-link
+  CAPSULE_REPO: replace-with-capsule-http-link
+  SATTOOLS_REPO:
+    RHEL6: replace-with-rhel6-http-link
+    RHEL7: replace-with-rhel7-http-link
+    RHEL8: replace-with-rhel8-http-link
   # Downstream Satellite-maintain repo
-  # SATMAINTENANCE_REPO: replace_with_sat_maintain_repo
+  SATMAINTENANCE_REPO: replace-with-sat-maintain-repo
   # Software Collection Repo
-  # RHSCL_REPO: replace_with_rhscl_repo
+  RHSCL_REPO: replace-with-rhscl-repo
   # Ansible 2.8 Repo
-  # ANSIBLE_REPO: replace_with_ansible_repo
+  ANSIBLE_REPO: replace-with-ansible-repo
   # Added swid_tools_repo for installing swid-tools and dnf-plugin-swidtags packages
   # which are essentially required for generating swid tags in RHEL8 content host.
-  # SWID_TOOLS_REPO: replace_with_swid_tools_repo
+  SWID_TOOLS_REPO: replace-with-swid-tools-repo
+  FAKE_REPO_ZOO3: replace-with-zoo-fedorapeople-repo
```

Comment:
parameter missing
* PUPPET4_REPO

Reply:
ack - this one at least isn't hit during test collection.

Reply:
yes, no impact, will cover it in another PR.